src/basic/math_trans/e043.poly1_FAC1.s

```diff
@@ -30,3 +33,6 @@ poly1_FAC1:
     ldy #$00                        ; FAC1 <- FAC1 * TEMPF1
     lda #TEMPF1
     jmp mul_MEM_FAC1
+
+
+}
```

Comment:
It would be great if we could have `jmp do_NOT_IMPLEMENTED_error` if `CONFIG_TRANSCENDENTAL_FUNCTIONS` is not defined.

Reply:
Good idea. I added this.